go.mod

```diff
@@ -0,0 +1,5 @@
+module go-protocol
+
+go 1.12
```

Comment:
Sollte das nicht bereits 1.13 sein? Soweit ich mich erinnern kann war das zu Beginn ein explizites requirement auf der neusten Version von Go zu entwickeln.

Reply:
Das ist m.E. die minimale Version. Die momentane Version in Ubuntu 19.04 ist 1.12 sowie auch in Fedora.

> The go directive in a go.mod file now indicates the version of the language used by the files within that module. It will be set to the current release (go 1.12) if no existing version is present. If the go directive for a module specifies a version newer than the toolchain in use, the go command will attempt to build the packages regardless, and will note the mismatch only if that build fails. (https://github.com/golang/go/issues/30791)